torch/csrc/autograd/python_function.cpp

```diff
@@ -44,19 +44,24 @@ PyObject *THPFunctionClass = nullptr;
 
 namespace torch { namespace autograd {
 
+void PyNode::throw_python_error() {
+  auto python_err = python_error();
```

Comment:
nit: auto&

Reply:
we are creating a new `python_error()` here, so it cann't be a lvalue reference.

Reply:
or just the conventional:
`    python_error err;
`